File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-list-all-worksheets-in-a-workbook_2.vb
    Private Sub ListSheets()
        Dim index As Integer = 0

        Dim rng As Excel.Range = Me.Application.Range("A1")

        For Each displayWorksheet As Excel.Worksheet In Me.Application.Worksheets
            rng.Offset(index, 0).Value2 = displayWorksheet.Name
            index += 1
        Next displayWorksheet
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-list-all-worksheets-in-a-workbook_2.cs
        private void ListSheets()
        {
            int index = 0;

            Excel.Range rng = this.Application.get_Range("A1");

            foreach (Excel.Worksheet displayWorksheet in this.Application.Worksheets)
            {
                rng.get_Offset(index, 0).Value2 = displayWorksheet.Name;
                index++;
            }
        }